msg.go

```diff
@@ -73,53 +71,23 @@ var (
 	ErrTime          error = &Error{err: "bad time"}      // ErrTime indicates a timing error in TSIG authentication.
 )
 
-// Id by default, returns a 16 bits random number to be used as a
-// message id. The random provided should be good enough. This being a
-// variable the function can be reassigned to a custom function.
-// For instance, to make it return a static value:
+// Id by default, returns a 16 bit random number (from crypto/rand) to
+// be used as a message id. This being a variable the function can be
```

Comment:
I think I'd rather this be worded something like: "Id by default returns a 16-bit random number to be used as a message id. The number is drawn from a cryptographically secure random number generator."